extra/modules/confiant-ad-quality/README.md

```diff
@@ -0,0 +1,72 @@
+# Overview
+
+This module allows obtains all bid responses for any given auction, send the results to Confiant server and find possible security issues inside bid responses.
```

Comment:
More like: "This module obtains all bid responses for any given auction, sends the results to Confiant server and finds possible security issues inside bid responses."